senders/twilio/voice.go

```diff
@@ -25,9 +28,14 @@ func (sender *twilioSenderVoice) SendEvents(events moira.NotificationEvents, con
 }
 
 func (sender *twilioSenderVoice) buildVoiceURL(trigger moira.TriggerData) string {
```

Comment:
Maybe it would be enough to rewrite twilio/buildVoiceURL() into something similar to webhook/buildRequestURL()
https://github.com/moira-alert/moira/blob/d796c99ea6a4ae2200ca539b1116fbad7f1b534b/senders/webhook/request.go#L52
to use "voiceurl" like here:
https://github.com/moira-alert/doc/blob/e7145befc6afcceeccf24fad3148163ba018eb39/notifier.example.yml#L74

Reply:
In my opinion the way URL is built here is ok. I do not thing that base URL will be changed frequently.